cli/package.json

```diff
@@ -106,6 +106,9 @@
     "spawn-mock": "1.0.0",
     "strip-ansi": "6.0.0"
   },
+  "peerDependencies": {
+    "@types/node": ">=16"
```

Comment:
Moving `@types/node` to be a peer dependency is a breaking change....


However, I took some time reading through the issue and then tracing through where `@types/node` was introduced and where it is used.

Like mentioned, this is used [in cy-http.d.ts](https://github.com/cypress-io/cypress/blob/f8cca7050012884097dedf830a12c034f10abbb4/cli/types/cy-http.d.ts#L6). This file notes:

>  This file should be deleted as soon as the serever
 * TODO: delete this file when ResolvedDevServerConfig.server is converted to closeServer

Which corresponds to [this note](https://github.com/cypress-io/cypress/pull/14479/files#diff-9fde1fb2f7eaba5ea49b585c811461495bfbba88e614e900a96863ecfe6b980aR5147) when this file was introduced.

Looking through the code base, there is no longer a need for this type and this file should have been cleaned up a while ago since the CT Server types has been updated to use a close func.

Instead of moving `@types/node` to a peer-dependency, we should be able to remove `cy-http.d.ts` file and the `@types/node` dependency.

Reply:
That's great, thanks @emilyrohrbough! I'll update this PR to make that change.